### PR TITLE
New ATLAS RAL proposal

### PR DESCRIPTION
**Pull request by StewMH**
Hi,

This is a new proposal for RAL ATLAS.

Cheers,
Stewart

### CODE REVIEW

**Comment by graeme-a-stewart** [comment]
I guess this needs discussed with the HSF GOSC coordinators, viz. @etejedor Sergei (sergeivgleyzer@gmail.com) and Antoine (perus@lal.in2p3.fr). Looks good to me though.

File: _gsocproposals/proposal_ATLASEigen.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## Requirements
+
+- C++
+- python
+- basic linear algebra
+
+## Mentors
+
+ * [Stewart Martin-Haugh](mailto:stewart.martin-haugh@stfc.ac.uk)
```
Comment (etejedor) [inline _gsocproposals/proposal_ATLASEigen.md]: Every project proposal needs at least two mentors. Please try to find another colleague that joins you for this project.

File: _gsocproposals/proposal_ATLASEigen.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+## Expected results
+
+A working implementation of symmetric matrices in Eigen, ready to be submitted as a patch for Eigen 
+
+## Requirements
+
+- C++
+- python
+- basic linear algebra
+
```
Comment (etejedor) [inline _gsocproposals/proposal_ATLASEigen.md]: Could you find one or two relevant links for the project where the student can get additional information? If so, please add a "Links" section with the links.

File: _gsocproposals/proposal_ATLASEigen.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+---
+title:  Faster matrix algebra for ATLAS
+layout: gsoc_proposal
+project: ATLAS
+organization:
+  - RAL
```
Comment (etejedor) [inline _gsocproposals/proposal_ATLASEigen.md]: RAL will need to be added as a new organisation under the CERN-HSF umbrella, could you please follow the instructions [here](http://hepsoftwarefoundation.org/gsoc/guideline.html) on that regard?

Comment (etejedor) [reply _gsocproposals/proposal_ATLASEigen.md]: Nevermind, just saw you already did that :)

File: _gsocproposals/proposal_ATLASEigen.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+---
```
Comment (etejedor) [inline _gsocproposals/proposal_ATLASEigen.md]: Hi Stewart,
We made some changes to the internal structure of the repository. Could you rebase the PR? For instance, your PR should go in another directory  (`_gsocproposals/2018`). Thank you!

**Comment by StewMH** [comment]
Sorry, I was on holiday while this happened. I am adding in a second supervisor (new PR).

**Pull request merged**